packages/xrpl/src/models/methods/accountObjects.ts

```diff
@@ -98,7 +55,7 @@ export interface AccountObjectsResponse extends BaseResponse {
      * Array of objects owned by this account. Each object is in its raw
      * ledger format.
      */
-    account_objects: AccountObject[]
+    account_objects: LedgerEntry[]
```

Comment:
Can you do something similar to AccountObjectsType and then add back AccountObject.  Small tweak will help intellisense.